falcon/routing/converters.py

```diff
@@ -83,20 +86,49 @@ def convert(self, value):
         except ValueError:
             return None
 
+        return self.validate_min_max_value(value)
+
+    def validate_min_max_value(self, value):
         if self._min is not None and value < self._min:
             return None
-
         if self._max is not None and value > self._max:
             return None
 
         return value
 
 
+class FloatConverter(IntConverter):
+    """Converts a field value to an float.
+
+    Identifier: `float`
+    Keyword Args:
+        min (float): Reject the value if it is less than this number.
+        max (float): Reject the value if it is greater than this number.
+    """
+
+    __slots__ = ('_min', '_max')
```

Comment:
Since this class extends int converter that already defines min and max slots should be empty in the subclass, since no new attribute is defined